README.md

```diff
@@ -1,4 +1,9 @@
-[![Build Status](https://travis-ci.org/GoogleCloudPlatform/forseti-security.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/forseti-security) [![codecov](https://codecov.io/gh/GoogleCloudPlatform/forseti-security/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/forseti-security)
+**branch: stable (2.0 production)** | **branch: dev (2.0 development)** | **branch: master (1.0 production)** | **branch: 1.0-dev** | **branch: forsetisecurity.org**
```

Comment:
I'm not a fan of these titles as you're attempting to add context to the name.

Let's use the literal branch name here and nothing else. Then readers can use the link below for more information.